tests/acceptance/30_generic_methods/file_check_FIFO_pipe.cf

```diff
@@ -0,0 +1,90 @@
+#########################################
+#
+# Test checking if a file is a FIFO or not 
+# 
+#########################################
+
+bundle common acc_path
+{
+  vars:
+    "root" string => getenv("NCF_TESTS_ACCEPTANCE", 1024);
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, default("$#this.promise_filename}") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  vars:
+    "tmp"            string => getenv("TEMP", 1024);
+    "file"           string => "${tmp}/test";
+    "file_canon"     string => canonify("${file}");
+
+    "FIFO_pipe"        string => "${tmp}/FIFO_pipe";
+    "FIFO_pipe_canon"  string => canonify("${FIFO_pipe}");
+
+    "file_list"       slist => { "${file}", "${FIFO_pipe}" };
+
+  files:
+    "${file}"
+      create => "true",
+      perms  => mog("000", "root", "0");
```

Comment:
You should probably just remove this "perms" line, it is setting *no* permissions (I assume it was copy pasted from a different acceptance test...). Using "000" for perms may break this test for unrelated reasons.

Omitting the perms body should just use sane defaults.